appengine/flexible/endpoints/README.md

```diff
@@ -93,3 +93,71 @@ To use the client ID for authentication:
 Now you can use the client ID to make requests to the API:
 
     $ python clients/google-id-token-client.py https://YOUR-PROJECT-ID.appspot.com YOUR-API-KEY /path/to/client-id.json
+
+### Using the App Engine default service account client (no key file needed)
+
+The App Engine default service account client demonstrates how to use the Google App Engine default service account to authenticate to endpoints.
+To use the client, you need to create a Google Cloud project as your client project.
+
+To use the App Engine default service account for authentication:
+
+1. Update the `gae_default_service_account`'s `x-issuer` and `x-jwks_uri` in `swagger.yaml` with your client project ID.
+2. Redeploy your server application.
+3. Update clients/service_to_service_gae_default/main.py, replace `YOUR-CLIENT-PROJECT-ID` and `YOUR-SERVER-PROJECT-ID` with your client project ID and your server project ID.
```

Comment:
In the samples, we want to show server to server call, so the client and server projects are different.